### ISSUE TITLE
NVDA Windows Chrome & Firefox - Play button is repeated multiple times when clicked.

### ISSUE THREAD

**Issue by waughcbw** [videojs/video.js#5023]
## Description
Once NVDA focus is on the play button and the button is clicked, NVDA repeats the state of the button multiple times (Chrome 6 times, Firefox 3 times).

## Steps to reproduce
With NVDA running, go to videojs.com and click the play button, either through enter key or mouse click.

### Expected
The state of the play button is stated only once when clicked.

### Actual
The state of the play button is stated 6 times in Windows Chrome and 3 times in Windows Firefox when clicked. e.g. When the play button is clicked in Windows Chrome, NVDA will read "Play, Play, Play, Play, Play, Play".

### Error output
N/A

#### videojs
Latest version on video.js website.
#### browsers
Chrome, Firefox
#### OSes
Windows 7 & 10

**Comment by jiangtiandao** [comment]
The interesting thing is what is announced is "Play Play lay Play Play Play"

**Comment by jiangtiandao** [comment]
http://jsbin.com/piqulisose/2/edit?html,js,console

**Comment by jiangtiandao** [comment]
I found a workaround by removing original play control button and implement one instead.
```HTML
<button class="vjs-play-control vjs-control vjs-button" type="button" aria-live="polite" title="Play" aria-disabled="false"><span aria-hidden="true" class="vjs-icon-placeholder"></span><span class="vjs-control-text">Play</span></button>
```
If we remove inner span will reduce to 4 times in Chrome.
If we remove the title attribute will reduce to 2 times  in Chrome.

**Comment by gkatsev** [comment]
Hey, thanks for the issue. We knew this has been happening but haven't had a chance to take a look at it. The title attribute was originally added so that you can visually read what a button that only had an icon but this is clearly causing issues. The plan is to go with removing the title attribute and add our own tooltip.

**Comment by alex-barstow** [comment]
This is reproducible without Video.js as well: http://solutions.brightcove.com/abarstow/BC-39639/nvda-test.html

**Comment by gkatsev** [comment]
This should be fixed as part of https://github.com/videojs/video.js/pull/5158
